packages/api-derive/src/staking/info.ts

```diff
@@ -203,8 +236,8 @@ export function info (api: ApiInterfaceRx): (_accountId: Uint8Array | string) =>
 
     return (
       api.queryMulti([
-        [api.query.staking.bonded, accountId], // try to map to controller
-        [api.query.staking.ledger, accountId] // try to map to stash
+        [api.query.staking.bonded, accountId.toString()], // try to map to controller
```

Comment:
Why the `toString` here? (And elsewhere)

Reply:
I was trying to figure out why the switchMap wouldn't go through and thought it was related to that :)